databricks_cli/pipelines/cli.py

```diff
@@ -186,20 +186,24 @@ def list_cli(api_client):
 @provide_api_client
 def reset_cli(api_client, pipeline_id):
     """
-    Resets a delta pipeline by truncating tables and creating new checkpoint folders so data is
+    [Deprecated] Use the "update --full-refresh" command instead.
+
+    Resets a pipeline by truncating tables and creating new checkpoint folders so data is
     reprocessed from scratch.
 
     Usage:
 
     databricks pipelines reset --pipeline-id 1234
     """
+    click.echo("DeprecationWarning: \"reset\" command is deprecated, " +
```

Comment:
For internationalization and style:
"reset" -> "the reset"
Omit "please"

Reply:
you mean <"reset" command> to <the "reset" command> right? 
ptal